lib/fuse_versionscript

```diff
@@ -205,8 +205,6 @@ FUSE_3.17 {
 FUSE_3.18 {
 	global:
 		fuse_req_is_uring;
-		fuse_uring_set_payload_allocator;
-		fuse_uring_get_req_payload_mr;
```

Comment:
For myself, some of the previous patches introduced `fuse_uring_set_payload_allocator`and `fuse_uring_get_req_payload_mr`. These won't be part of the initial API and maybe won't be needed at all. I need to see about it.